app/models/patient.rb

```diff
@@ -188,6 +188,10 @@ def call_result=(new_call_result)
     super(new_call_result)
   end
 
+  def prescribed_drugs(date: Date.today)
+    prescription_drugs.prescribed_as_on(date: date)
```

Comment:
```suggestion
    prescription_drugs.prescribed_as_of(date: date)
```

Reply:
Could `active_on` work too? Just throwing it into the mix, I'll let you pick the one you think reads best.